### PR TITLE
Add `log_url` table to MariaDB

### LINKED ISSUE #606: Add `log_url` table to mariadb
the `kubearchive.sql` file for mariadb does not have a `log_url` table. We should add this table to mariadb

### PR DESCRIPTION
**Pull request by skoved**
## Which Issue(s) this Pull Request Resolves

Resolves #606 

## Release Note

```release-note
NONE
```

### CODE REVIEW

File: integrations/database/mariadb/kubearchive.sql [hunk 1]
```diff
@@ -46,6 +46,19 @@ CREATE TABLE `resource` (
...
+-- Table structure for table `log_urls`
+--
+
+DROP TABLE IF EXISTS `log_url`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `log_url` (
+  `uuid` uuid NOT NULL,
+  `url` text NOT NULL,
+  `container_name` text NOT NULL,
```
Comment (maruiz93) [inline integrations/database/mariadb/kubearchive.sql:60]: can we also add `created_at` and `updated_at` like we have in the `resource` table?

**Review by maruiz93 — CHANGES_REQUESTED** [review]

File: integrations/database/mariadb/kubearchive.sql (hunk 1 above)
Comment (skoved) [reply integrations/database/mariadb/kubearchive.sql:60]: done

**Review by ggallen — APPROVED** [review]

**Review by mafh314 — APPROVED** [review]

**Comment by skoved** [comment]
rebasing to get changes from `main`

**Comment by skoved** [comment]
@maruiz93 PTAL when you have time. This is low priority though so no rush :smiley:

**Comment by maruiz93** [comment]
> @maruiz93 PTAL when you have time. This is low priority though so no rush 😃

Ops! Sorry I totally forgot I have this PR pending to review, thank you for the reminder :)

File: integrations/database/mariadb/kubearchive.sql [hunk 2]
```diff
@@ -46,6 +46,21 @@ CREATE TABLE `resource` (
...
+  `created_at` timestamp NOT NULL DEFAULT current_timestamp(),
+  `updated_at` timestamp NOT NULL DEFAULT current_timestamp()
```
Comment (maruiz93) [inline integrations/database/mariadb/kubearchive.sql:62]: I miss the trigger, we have one for the `resource` table but it is easily missed as the maria db statements begin with `/*!` to specify the version accepted and they "seem" comments.

The trigger is in line 75, we should add a similar one for the `log_url.updated_at` field

**Review by maruiz93 — CHANGES_REQUESTED** [review]
I left a comment regarding the trigger to automatically update the `updated_at` added field for the `log_url` table.

**Review by maruiz93 — APPROVED** [review]

**Comment by skoved** [comment]
rebasing from `main` so this PR can be merged

**Pull request merged**